python/docs/librmm_docs/index.rst

```diff
@@ -0,0 +1,29 @@
+.. rmm documentation master file, created by
+   sphinx-quickstart on Thu Nov 19 13:16:00 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+librmm Documentation
+====================
```

Comment:
This heading makes the breadcrumbs deeply nested. The navigation is awkward as a result.

![image](https://github.com/rapidsai/rmm/assets/3943761/90a21f46-e92e-48a5-ba09-664a38b7472a)

Can we eliminate this level of the hierarchy entirely, so that it's something more like `Home / rmm C++ / API Reference / Memory Resources`? See if you can fuse this page's `toctree` into `python/docs/cpp.rst` instead.